src/test/java/seedu/exercise/storage/JsonAdaptedExerciseTest.java

```diff
@@ -1,21 +1,21 @@
-package seedu.address.storage;
+package seedu.exercise.storage;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static seedu.address.storage.JsonAdaptedExercise.MISSING_FIELD_MESSAGE_FORMAT;
-import static seedu.address.testutil.Assert.assertThrows;
-import static seedu.address.testutil.TypicalExercises.SWIM;
+import static seedu.exercise.storage.JsonAdaptedExercise.MISSING_FIELD_MESSAGE_FORMAT;
+import static seedu.exercise.testutil.Assert.assertThrows;
+import static seedu.exercise.testutil.TypicalExercises.SWIM;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import org.junit.jupiter.api.Test;
 
-import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.exercise.Calories;
-import seedu.address.model.exercise.Date;
-import seedu.address.model.exercise.Name;
-import seedu.address.model.exercise.Quantity;
+import seedu.exercise.commons.exceptions.IllegalValueException;
+import seedu.exercise.model.exercise.Calories;
+import seedu.exercise.model.exercise.Date;
+import seedu.exercise.model.exercise.Name;
+import seedu.exercise.model.exercise.Quantity;
 
 public class JsonAdaptedExerciseTest {
     private static final String INVALID_NAME = "R@chel";
```

Comment:
May want to change the values of these variables to better reflect their names.
eg "R@chel" to "D@nce"

Reply:
done